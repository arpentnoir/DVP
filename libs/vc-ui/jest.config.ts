/* eslint-disable */
export default {
  displayName: 'vc-ui',
  preset: '../../jest.preset.js',
  setupFiles: ['./jest.setup.ts'],
  transform: {
    '^.+\\.[tj]sx?$': 'babel-jest',
  },
  moduleFileExtensions: ['ts', 'tsx', 'js', 'jsx'],
  coverageDirectory: '../../coverage/libs/vc-ui',
  moduleNameMapper: {
    '^react-pdf': 'react-pdf/dist/umd/entry.jest',
  },
};
