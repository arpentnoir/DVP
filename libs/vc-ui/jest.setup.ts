import 'jest-canvas-mock';
import '@testing-library/jest-dom';

const mockClipboard = {
  writeText: jest.fn(),
};

(global.navigator.clipboard as any) = mockClipboard;
