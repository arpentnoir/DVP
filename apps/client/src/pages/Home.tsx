import { Card, Text } from '@dvp/vc-ui';
import NavigateNextIcon from '@mui/icons-material/NavigateNext';
import { Box, ButtonBase, Grid, Stack, useTheme } from '@mui/material';
import { useNavigate } from 'react-router-dom';
import { ROUTES } from '../constants';
import { BaseLayout } from '../layouts';

// TODO: Replace with real data
const actions = [
  {
    name: 'Lorem',
    text: 'It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout.',
  },
  {
    name: 'Ipsum',
    text: 'It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout.',
  },
  {
    name: 'Dolor',
    text: 'It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout.',
  },
  {
    name: 'Amet',
    text: 'It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout.',
  },
];

export const Home = () => {
  const theme = useTheme();
  const navigate = useNavigate();

  return (
    <BaseLayout
      title="Digital Verification Platform"
      sx={{
        paddingX: 0,
        paddingY: 0,
        backgroundColor: theme.palette.common.backgroundBlue,
        display: 'flex',
        justifyContent: 'center',
        maxWidth: '100%',
      }}
    >
      <Grid container height={'100%'} maxWidth={'1140px'}>
        <Stack>
          <Stack
            direction={'row'}
            sx={{
              height: 'calc(50vw * 0.62)',
              minHeight: { xs: '370px', md: '410px' },
              maxHeight: '450px',
              width: '100%',
            }}
          >
            <Grid item xs={12} sm={6}>
              <Stack
                padding={{
                  xs: '60px 20px 0px 20px',
                  sm: '40px 30px',
                  md: '60px 0px 0px 30px',
                }}
              >
                <Box>
                  <Text variant="h1" fontWeight="bold" color="white">
                    DIGITAL VERIFICATION
                  </Text>
                  <Text variant="h1" fontWeight="bold" color="white">
                    PLATFORM
                  </Text>
                </Box>
                <Text color="white" paddingTop="12px">
                  Lorem ipsum dolar sit amet consectetur
                </Text>
                <Stack padding="8px 10px 0 0">
                  <ButtonBase onClick={() => navigate(ROUTES.VERIFY)}>
                    <Stack
                      width="100%"
                      direction="row"
                      justifyContent="space-between"
                      alignItems="center"
                      sx={{
                        borderBottom: '1px solid white',
                        padding: '16px 20px',
                      }}
                    >
                      <Text color="white" fontWeight="bold" variant="body1">
                        Verify a document
                      </Text>
                      <NavigateNextIcon
                        sx={{ color: theme.palette.common.featureGold }}
                      />
                    </Stack>
                  </ButtonBase>
                  <ButtonBase>
                    <Stack
                      width="100%"
                      direction="row"
                      justifyContent="space-between"
                      alignItems="center"
                      sx={{
                        borderBottom: '1px solid white',
                        padding: '16px 20px',
                      }}
                    >
                      <Text color="white" fontWeight="bold" variant="body1">
                        Help and support
                      </Text>
                      <NavigateNextIcon
                        sx={{ color: theme.palette.common.featureGold }}
                      />
                    </Stack>
                  </ButtonBase>
                </Stack>
              </Stack>
            </Grid>
            <Grid
              item
              sm={6}
              sx={{
                display: {
                  xs: 'none',
                  sm: 'block',
                  background: 'url("assets/home-background.jpg")',
                  backgroundPosition: 'bottom right',
                  backgroundSize: 'cover',
                  backgroundRepeat: 'no-repeat',
                  overflow: 'hidden',
                },
                position: 'absolute',
                width: '100%',
                right: 0,
                height: 'calc(50vw * 0.62)',
                minHeight: { xs: '370px', md: '410px' },
                maxHeight: '450px',
              }}
            >
              <Box
                sx={{
                  background: theme.palette.common.backgroundBlue,
                  position: 'relative',
                  top: '-100px',
                  left: '-120px',
                  width: '200px',
                  height: 'calc(100% + 300px)',
                  transform: 'rotate(-15deg)',
                }}
              />
            </Grid>
          </Stack>
          <Grid
            container
            spacing="30px"
            padding={{
              xs: '0px 40px 100px 40px',
              sm: '60px 40px',
            }}
          >
            {actions.map((action) => (
              <Grid item lg={3} md={4} sm={6} xs={12} key={action.name}>
                <Card
                  sx={{ flex: 1, padding: '16px' }}
                  name={action.name}
                  text={action.text}
                  handleAction={() => null}
                  actionLabel="Action"
                />
              </Grid>
            ))}
          </Grid>
        </Stack>
      </Grid>
    </BaseLayout>
  );
};
