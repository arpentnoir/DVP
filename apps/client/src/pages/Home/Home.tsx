import { Card, Text } from '@dvp/vc-ui';
import NavigateNextIcon from '@mui/icons-material/NavigateNext';
import { Box, ButtonBase, Grid, Stack, useTheme } from '@mui/material';
import { useNavigate } from 'react-router-dom';
import { ROUTES } from '../../constants';
import { BaseLayout } from '../../layouts';

// TODO: Replace with real data
const actions = [
  {
    name: 'LOREM',
    text: 'It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout.',
  },
  {
    name: 'IPSUM',
    text: 'It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout.',
  },
  {
    name: 'DOLOR',
    text: 'It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout.',
  },
  {
    name: 'LOREM',
    text: 'It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout.',
  },
  {
    name: 'IPSUM',
    text: 'It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout.',
  },
  {
    name: 'DOLOR',
    text: 'It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout.',
  },
];

export const Home = () => {
  const theme = useTheme();
  const navigate = useNavigate();

  return (
    <BaseLayout
      title="Digital Verification Platform"
      sx={{
        paddingX: 0,
        paddingY: 0,
        backgroundColor: theme.palette.common.backgroundBlue,
        display: 'flex',
        justifyContent: 'center',
        flexDirection: 'column',
        maxWidth: '100%',
      }}
    >
      <Box width="100%" sx={{ display: 'flex', justifyContent: 'center' }}>
        <Grid container maxWidth={'1140px'}>
          <Stack
            direction={'row'}
            sx={{
              height: 'calc(50vw * 0.62)',
              minHeight: { xs: '370px', md: '410px' },
              maxHeight: '450px',
              width: '100%',
            }}
          >
            <Grid item xs={12} md={6}>
              <Stack
                padding={{
                  xs: '60px 20px 0px 20px',
                  sm: '40px 30px',
                  md: '60px 0px 0px 30px',
                }}
              >
                <Box>
                  <Text variant="h1" fontWeight="bold" color="white">
                    DIGITAL VERIFICATION
                  </Text>
                  <Text variant="h1" fontWeight="bold" color="white">
                    PLATFORM
                  </Text>
                </Box>
                <Text color="white" paddingTop="12px">
                  Lorem ipsum dolar sit amet consectetur
                </Text>
                <Stack padding="8px 10px 0 0">
                  <ButtonBase onClick={() => navigate(ROUTES.VERIFY)}>
                    <Stack
                      width="100%"
                      direction="row"
                      justifyContent="space-between"
                      alignItems="center"
                      sx={{
                        borderBottom: '1px solid white',
                        padding: '16px 20px',
                        ':hover': {
                          borderColor: theme.palette.common.featureGold,
                        },
                      }}
                    >
                      <Text color="white" fontWeight="bold" variant="body1">
                        Verify a document
                      </Text>
                      <NavigateNextIcon
                        sx={{ color: theme.palette.common.featureGold }}
                      />
                    </Stack>
                  </ButtonBase>
                  <ButtonBase>
                    <Stack
                      width="100%"
                      direction="row"
                      justifyContent="space-between"
                      alignItems="center"
                      sx={{
                        borderBottom: '1px solid white',
                        padding: '16px 20px',
                        ':hover': {
                          borderColor: theme.palette.common.featureGold,
                        },
                      }}
                    >
                      <Text color="white" fontWeight="bold" variant="body1">
                        Help and support
                      </Text>
                      <NavigateNextIcon
                        sx={{ color: theme.palette.common.featureGold }}
                      />
                    </Stack>
                  </ButtonBase>
                </Stack>
              </Stack>
            </Grid>
            <Grid
              item
              md={6}
              sx={{
                display: {
                  xs: 'none',
                  md: 'block',
                  background: 'url("assets/home-background.jpg")',
                  backgroundPosition: 'bottom right',
                  backgroundSize: 'cover',
                  backgroundRepeat: 'no-repeat',
                  overflow: 'hidden',
                },
                position: 'absolute',
                width: '100%',
                right: 0,
                height: 'calc(50vw * 0.62)',
                minHeight: { xs: '370px', md: '410px' },
                maxHeight: '450px',
              }}
            >
              <Box
                sx={{
                  height: 0,
                  width: 0,
                  borderBottom: `700px solid ${theme.palette.common.backgroundBlue}`,
                  borderRight: '200px solid transparent',
                }}
              />
            </Grid>
          </Stack>
        </Grid>
      </Box>
      <Box
        sx={{
          backgroundColor: theme.palette.common.neutralGrey,
          display: 'flex',
          justifyContent: 'center',
        }}
      >
        <Grid
          container
          maxWidth={'1140px'}
          padding={{
            xs: '20px',
            sm: '60px 40px',
          }}
        >
          {actions.map((action) => (
            <Grid item md={4} sm={6} xs={12} key={action.name}>
              <Card
                sx={{
                  flex: 1,
                  padding: '16px',
                  borderRadius: 0,
                  borderColor: theme.palette.common.neutralGrey,
                  borderWidth: '1px',
                  borderStyle: 'solid',
                  ':hover': {
                    backgroundColor: theme.palette.common.backgroundBlue,
                    color: 'white',
                    borderColor: theme.palette.common.backgroundBlue,
                  },
                  cursor: 'pointer',
                }}
                elevation={0}
                name={action.name}
                text={action.text}
                headerAction={
                  <NavigateNextIcon
                    sx={{ color: theme.palette.common.featureGold }}
                  />
                }
              />
            </Grid>
          ))}
        </Grid>
      </Box>
    </BaseLayout>
  );
};
