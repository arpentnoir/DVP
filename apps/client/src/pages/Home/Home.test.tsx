import { render } from '@dvp/vc-ui';
import { Home } from './Home';

describe('Home', () => {
  it('should render correctly', () => {
    const { baseElement } = render(<Home />);

    expect(baseElement).toMatchSnapshot();
  });
});
