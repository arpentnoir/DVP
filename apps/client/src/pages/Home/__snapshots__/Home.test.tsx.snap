// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render correctly 1`] = `
<body>
  <div>
    <div
      class="MuiContainer-root MuiContainer-disableGutters css-8qkppm-MuiContainer-root"
    >
      <div
        class="MuiBox-root css-1ykdma4"
      >
        <div
          class="MuiGrid-root MuiGrid-container css-12zsqbf-MuiGrid-root"
        >
          <div
            class="css-7zlskd-MuiStack-root"
          >
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-6 css-1fyyp8j-MuiGrid-root"
            >
              <div
                class="css-hwbtf3-MuiStack-root"
              >
                <div
                  class="MuiBox-root css-0"
                >
                  <h1
                    class="MuiTypography-root MuiTypography-h1 css-wc1p7j-MuiTypography-root"
                    tabindex="0"
                  >
                    DIGITAL VERIFICATION
                  </h1>
                  <h1
                    class="MuiTypography-root MuiTypography-h1 css-wc1p7j-MuiTypography-root"
                    tabindex="0"
                  >
                    PLATFORM
                  </h1>
                </div>
                <p
                  class="MuiTypography-root MuiTypography-body2 css-5wlk1t-MuiTypography-root"
                  tabindex="0"
                >
                  Lorem ipsum dolar sit amet consectetur
                </p>
                <div
                  class="css-124fyjl-MuiStack-root"
                >
                  <button
                    class="MuiButtonBase-root css-10d1a0h-MuiButtonBase-root"
                    tabindex="0"
                    type="button"
                  >
                    <div
                      class="css-1qqnyiw-MuiStack-root"
                    >
                      <p
                        class="MuiTypography-root MuiTypography-body1 css-10rr26n-MuiTypography-root"
                        tabindex="0"
                      >
                        Verify a document
                      </p>
                      <svg
                        aria-hidden="true"
                        class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                        data-testid="NavigateNextIcon"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M10 6 8.59 7.41 13.17 12l-4.58 4.59L10 18l6-6z"
                        />
                      </svg>
                    </div>
                    <span
                      class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                    />
                  </button>
                  <button
                    class="MuiButtonBase-root css-10d1a0h-MuiButtonBase-root"
                    tabindex="0"
                    type="button"
                  >
                    <div
                      class="css-1qqnyiw-MuiStack-root"
                    >
                      <p
                        class="MuiTypography-root MuiTypography-body1 css-10rr26n-MuiTypography-root"
                        tabindex="0"
                      >
                        Help and support
                      </p>
                      <svg
                        aria-hidden="true"
                        class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                        data-testid="NavigateNextIcon"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M10 6 8.59 7.41 13.17 12l-4.58 4.59L10 18l6-6z"
                        />
                      </svg>
                    </div>
                    <span
                      class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                    />
                  </button>
                </div>
              </div>
            </div>
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-md-6 css-ovf0jd-MuiGrid-root"
            >
              <div
                class="MuiBox-root css-115ea40"
              />
            </div>
          </div>
        </div>
      </div>
      <div
        class="MuiBox-root css-1l4w6pd"
      >
        <div
          class="MuiGrid-root MuiGrid-container css-13jiec0-MuiGrid-root"
        >
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4 css-11l5t4l-MuiGrid-root"
          >
            <div
              class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation0 MuiCard-root css-1tm4xf1-MuiPaper-root-MuiCard-root"
            >
              <div
                class="MuiCardContent-root css-46bh2p-MuiCardContent-root"
              >
                <div
                  class="css-3y1137-MuiStack-root"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body1 css-ndaqpv-MuiTypography-root"
                    tabindex="0"
                  >
                    LOREM
                  </p>
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                    data-testid="NavigateNextIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M10 6 8.59 7.41 13.17 12l-4.58 4.59L10 18l6-6z"
                    />
                  </svg>
                </div>
                <p
                  class="MuiTypography-root MuiTypography-body2 css-ik0l5y-MuiTypography-root"
                  tabindex="0"
                >
                  It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout.
                </p>
              </div>
            </div>
          </div>
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4 css-11l5t4l-MuiGrid-root"
          >
            <div
              class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation0 MuiCard-root css-1tm4xf1-MuiPaper-root-MuiCard-root"
            >
              <div
                class="MuiCardContent-root css-46bh2p-MuiCardContent-root"
              >
                <div
                  class="css-3y1137-MuiStack-root"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body1 css-ndaqpv-MuiTypography-root"
                    tabindex="0"
                  >
                    IPSUM
                  </p>
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                    data-testid="NavigateNextIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M10 6 8.59 7.41 13.17 12l-4.58 4.59L10 18l6-6z"
                    />
                  </svg>
                </div>
                <p
                  class="MuiTypography-root MuiTypography-body2 css-ik0l5y-MuiTypography-root"
                  tabindex="0"
                >
                  It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout.
                </p>
              </div>
            </div>
          </div>
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4 css-11l5t4l-MuiGrid-root"
          >
            <div
              class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation0 MuiCard-root css-1tm4xf1-MuiPaper-root-MuiCard-root"
            >
              <div
                class="MuiCardContent-root css-46bh2p-MuiCardContent-root"
              >
                <div
                  class="css-3y1137-MuiStack-root"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body1 css-ndaqpv-MuiTypography-root"
                    tabindex="0"
                  >
                    DOLOR
                  </p>
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                    data-testid="NavigateNextIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M10 6 8.59 7.41 13.17 12l-4.58 4.59L10 18l6-6z"
                    />
                  </svg>
                </div>
                <p
                  class="MuiTypography-root MuiTypography-body2 css-ik0l5y-MuiTypography-root"
                  tabindex="0"
                >
                  It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout.
                </p>
              </div>
            </div>
          </div>
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4 css-11l5t4l-MuiGrid-root"
          >
            <div
              class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation0 MuiCard-root css-1tm4xf1-MuiPaper-root-MuiCard-root"
            >
              <div
                class="MuiCardContent-root css-46bh2p-MuiCardContent-root"
              >
                <div
                  class="css-3y1137-MuiStack-root"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body1 css-ndaqpv-MuiTypography-root"
                    tabindex="0"
                  >
                    LOREM
                  </p>
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                    data-testid="NavigateNextIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M10 6 8.59 7.41 13.17 12l-4.58 4.59L10 18l6-6z"
                    />
                  </svg>
                </div>
                <p
                  class="MuiTypography-root MuiTypography-body2 css-ik0l5y-MuiTypography-root"
                  tabindex="0"
                >
                  It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout.
                </p>
              </div>
            </div>
          </div>
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4 css-11l5t4l-MuiGrid-root"
          >
            <div
              class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation0 MuiCard-root css-1tm4xf1-MuiPaper-root-MuiCard-root"
            >
              <div
                class="MuiCardContent-root css-46bh2p-MuiCardContent-root"
              >
                <div
                  class="css-3y1137-MuiStack-root"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body1 css-ndaqpv-MuiTypography-root"
                    tabindex="0"
                  >
                    IPSUM
                  </p>
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                    data-testid="NavigateNextIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M10 6 8.59 7.41 13.17 12l-4.58 4.59L10 18l6-6z"
                    />
                  </svg>
                </div>
                <p
                  class="MuiTypography-root MuiTypography-body2 css-ik0l5y-MuiTypography-root"
                  tabindex="0"
                >
                  It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout.
                </p>
              </div>
            </div>
          </div>
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4 css-11l5t4l-MuiGrid-root"
          >
            <div
              class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation0 MuiCard-root css-1tm4xf1-MuiPaper-root-MuiCard-root"
            >
              <div
                class="MuiCardContent-root css-46bh2p-MuiCardContent-root"
              >
                <div
                  class="css-3y1137-MuiStack-root"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body1 css-ndaqpv-MuiTypography-root"
                    tabindex="0"
                  >
                    DOLOR
                  </p>
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                    data-testid="NavigateNextIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M10 6 8.59 7.41 13.17 12l-4.58 4.59L10 18l6-6z"
                    />
                  </svg>
                </div>
                <p
                  class="MuiTypography-root MuiTypography-body2 css-ik0l5y-MuiTypography-root"
                  tabindex="0"
                >
                  It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout.
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</body>
`;
